framework/mutex/MysqlMutex.php

```diff
@@ -86,13 +90,33 @@ protected function releaseLock($name)
     {
         return $this->db->useMaster(function ($db) use ($name) {
             /** @var \yii\db\Connection $db */
-            return (bool) $db->createCommand(
-                'SELECT RELEASE_LOCK(SUBSTRING(CONCAT(:prefix, :name), 1, 64))',
-                [':name' => $this->hashLockName($name), ':prefix' => $this->keyPrefix]
+            $nameData = $this->prepareName();
+            return (bool)$db->createCommand(
+                'SELECT RELEASE_LOCK(' . $nameData[0] . '), :prefix',
+                array_merge(
+                    [':name' => $this->hashLockName($name), ':prefix' => $this->keyPrefix],
+                    $nameData[1]
+                )
             )->queryScalar();
         });
     }
 
+    /**
+     * Prepare lock name
+     * @return array expression and params
+     * @since 2.0.48
+     */
+    private function prepareName()
```

Comment:
Should be `protected` else overriding methods where it's used will be very hard.

Reply:
modified to protected